services/QuillLMS/spec/factories/teacher_notification_setting.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :teacher_notification_setting, class: TeacherNotificationSetting do
```

Comment:
`class: TeacherNotificationSetting` should not be needed as factory bot can infer from filename.